labman/db/plate.py

```diff
@@ -219,16 +223,21 @@ def list_plates(plate_types=None, only_quantified=False,
             if only_quantified:
                 sql_join = ("JOIN qiita.concentration_calculation "
```

Comment:
Now that we're adding `sql_studies`, is `sql_join` the right name here? (just asking for clarity/future readability)

Reply:
I think it is still accurate, since it is adding more tables to the JOIN part of the query